hammerspoon/windows.lua

```diff
@@ -154,6 +154,23 @@ function hs.window.centerWithFullHeight(win)
   win:setFrame(f)
 end
 
+-- +--------------+
+-- |  |--------|  |
+-- |  |  HERE  |  |
+-- |  |--------|  |
+-- +---------------+
+function hs.window.middle(win)
+  local f = win:frame()
+  local screen = win:screen()
+  local max = screen:fullFrame()
+
+  f.w = 1280
+  f.h = 1024
```

Comment:
@bkeepers: Do you find it useful to limit the height of the window?

I ask because I definitely appreciate being able to limit the width of the window (which I often do with the `centerWithFullHeight` function below), but I can't think of a scenario in my workflow where I want to limit the height to less than the height of my current display.

https://github.com/jasonrudolph/keyboard/blob/4f7a69f9bf1fb233e5bc98c1db3e45de2d22c0ff/hammerspoon/windows.lua#L140-L155